autoload.rb

```diff
@@ -0,0 +1,10 @@
+require 'pry'
+require 'yaml'
+require 'date'
+require './modules/validation.rb'
+require './modules/storage.rb'
+require './entities/reader.rb'
+require './entities/book.rb'
+require './entities/order.rb'
+require './entities/author.rb'
+require './entities/library.rb'
```

Comment:
1) add `require './modules/yaml.rb'`
2) create file with path './modules/yaml.rb'
3) add following to that file:
```rb
module YAML
  class << self
    alias load unsafe_load if YAML.respond_to? :unsafe_load
  end
end
```